tests/integration/test_testing.py

```diff
@@ -134,13 +138,33 @@ class TestElasticFixture(SparklyGlobalSessionTest):
     ]
 
     def test_elastic_fixture(self):
-        df = self.spark.read_ext.by_url('elastic://elastic.docker/sparkly_test_fixture/test?'
+        df = self.spark.read_ext.by_url('elastic://elastic6.docker/sparkly_test_fixture/test?'
                                      'es.read.metadata=false')
         self.assertDataFrameEqual(df, [
             {'name': 'John', 'age': 56},
         ])
 
 
+class TestElastic7Fixture(SparklyGlobalSessionTest):
+
+    session = SparklyTestSessionWithES7
+
+    class_fixtures = [
+        ElasticFixture(
+            'elastic7.docker',
+            'sparkly_test_fixture',
+            None,
+            absolute_path(__file__, 'resources', 'test_fixtures', 'mapping.json'),
+            absolute_path(__file__, 'resources', 'test_fixtures', 'data_for_es7.json'),
+        )
+    ]
+
+    def test_elastic_fixture(self):
+        df = self.spark.read_ext.by_url('elastic://elastic7.docker/sparkly_test_fixture?'
```

Comment:
Let's indent this as
```
df = self.spark.read_ext.by_url(
    'elastic://....',
)
```
It's easier to read as it keeps the whole URL in one line, and the closing paren very visibly closes the block